ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/Payload.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright contributors to Hyperledger Besu.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.ethereum.core;
+
+import java.util.Objects;
+
+import org.apache.tuweni.bytes.Bytes;
+
+public class Payload {
+  private final Bytes payloadBytes;
+
+  private Long zeroBytes = null;
+
+  public Payload(final Bytes payloadBytes) {
+    this.payloadBytes = payloadBytes;
+  }
+
+  public Bytes getPayload() {
+    return payloadBytes;
+  }
+
+  public long zeroBytes() {
```

Comment:
sorry another naming thing, suggest getZeroBytesCount()

Reply:
Makes sense, I've renamed it